moveit_planners/ompl/ompl_interface/src/ompl_interface.cpp

```diff
@@ -183,33 +183,33 @@ void ompl_interface::OMPLInterface::loadPlannerConfigurations()
     std::map<std::string, std::string> specific_group_params;
     for (const std::string& k : KNOWN_GROUP_PARAMS)
     {
-      if (nh_.hasParam(group_name + "/" + k))
+      if (nh_.hasParam(group_name + "/" + k))  // NOLINT(performance-inefficient-string-concatenation)
       {
         std::string value;
-        if (nh_.getParam(group_name + "/" + k, value))
+        if (nh_.getParam(group_name + "/" + k, value))  // NOLINT(performance-inefficient-string-concatenation)
         {
           if (!value.empty())
             specific_group_params[k] = value;
           continue;
         }
 
         double value_d;
-        if (nh_.getParam(group_name + "/" + k, value_d))
+        if (nh_.getParam(group_name + "/" + k, value_d))  // NOLINT(performance-inefficient-string-concatenation)
         {
           // convert to string using no locale
           specific_group_params[k] = moveit::core::toString(value_d);
           continue;
         }
 
         int value_i;
-        if (nh_.getParam(group_name + "/" + k, value_i))
```

Comment:
Fix this!

Reply:
True, I actually noticed this first scrolling through, but then I got distracted by the general nolints :)